error_checks.c

```diff
@@ -43,13 +45,29 @@ bool check_battery_voltage_error(void){    //returns mV
         can_msg_t error_msg;
         build_board_stat_msg(timestamp, error_code, batt_data, 2, &error_msg);
         txb_enqueue(&error_msg);
+
+        // main loop should check this and open the vent valve if needed
+        if (batt_voltage_mV < VENT_BATT_UNDERVOLTAGE_PANIC_THRESHOLD_mV) {
```

Comment:
I feel like there should be some amount of hysteresis on this, something like "we need to read 10 good battery voltages in a row before we consider the voltage to no longer be critical". I worry that the battery will get to exactly the threshold voltage, and then noise on the ADC will cause us to toggle the vent every quarter second.

There's no major downside to toggling it like that (bigger current draw, maybe stressing the MOSFETs a bit), but still.